js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
```

Comment:
В camelCase. Константы пишутся в верхнем регистре только если эти данные не меняются ДО выполнения кода. А тут ты вычисляешь значение и динамически его присваивашь во время первого выполнения. Да, ссылка на объект не будет меняться, это константа, но это не предопределенная константа, это не данные "извне" а результат выполнения.
```suggestion
const samplePicture = document.querySelector('#picture').content.cloneNode(true);
```

Reply:
+